src/aspire/source/image.py

```diff
@@ -1005,7 +1021,7 @@ def save(
             starfile_filepath,
             filename_indices=filename_indices,
             batch_size=batch_size,
-            overwrite=overwrite,
```

Comment:
Can we just add an `elif overwrite is None: overwrite = False` above? This way we don't have to think about what `bool(None)` evaluates to…

Reply:
Good call. Removed `bool(overwrite)` in the three places it occurred. Thanks.